components/dashboard/src/components/forms/CheckboxInputField.tsx

```diff
@@ -56,7 +56,7 @@ export const CheckboxInput: FC<CheckboxInputProps> = ({
             <input
                 type="checkbox"
                 className={classNames(
-                    "h-4 w-4 mt-0.5 rounded cursor-pointer border-2 dark:filter-invert",
+                    "h-4 w-4 mt-1 rounded cursor-pointer border-2 dark:filter-invert",
```

Comment:
**issue(non-blocking)**: This is out of the scope of this PR, but the checkbox is using some unnecessary shades of gray, especially on the clicked state, see `2`. Cc @selfcontained 

| 1 | 2 | 3 | 4 |
|-|-|-|-|
| <img width="380" alt="Frame 1374" src="https://user-images.githubusercontent.com/120486/228931711-c225d3d5-dc73-4c96-8d1b-b2edabc0769a.png"> | <img width="380" alt="Frame 1375" src="https://user-images.githubusercontent.com/120486/228931721-ca31f029-3700-4557-b156-9a78cf9fbac4.png"> | <img width="380" alt="Frame 1376" src="https://user-images.githubusercontent.com/120486/228931730-7612306b-4576-4c0e-bf45-ca596eefd624.png"> | <img width="380" alt="Frame 1377" src="https://user-images.githubusercontent.com/120486/228931736-c63c968a-655b-4efe-842f-52ffd2ec1d6f.png"> |

Reply:
@Devansu-Yadav Ideally, this would not have the darker border on steps 2 and 4. If this is trivial, could you include this here? Otherwise, it should be fine to address separately. Your call. 🏓

Reply:
@gtsiolis Sure, it's a trivial change, and I'll address it. Does this border on the checkbox square look fine in dark mode or do I need to address it as well?

Reply:
Thanks @Devansu-Yadav! On the dark theme it's less noticeable but it also needs an update, see the different border color on :b:, not the focus rings. 🌔
 
| :a: | 🅱️  |
|-|-|
| <img width="725" alt="Frame 1369" src="https://user-images.githubusercontent.com/120486/229064647-4b2bf4f9-dbdc-4105-95a4-cb3c6bda0f18.png"> | <img width="725" alt="Frame 1370" src="https://user-images.githubusercontent.com/120486/229064653-14111878-f07a-46d9-8a2e-653139889668.png"> |

Reply:
@gtsiolis I have pushed a commit to address both of these.

Reply:
@Devansu-Yadav Perfection. ✨ 

![perfection](https://user-images.githubusercontent.com/120486/229114118-7c16ec14-aff8-457d-81db-caa95d15020a.gif)